src/VirtualGrid.vue

```diff
@@ -424,6 +436,7 @@ export default class VirtualGrid<P> extends Vue {
             <div
                 v-for="item in renderData.cellsToRender"
                 :key="item.id"
+                class="grid-item-wraper"
```

Comment:
what is the reason for this change?

Reply:
This will make it easier to add some styles to each item (For example, adjusting the zIndex when hovering). Of course, it can be done without it, but it's easier.

Reply:
```suggestion
                class="grid-item-wrapper"
```
ok works for me